### PR TITLE
Add remote control command next track and previous track for iOS

### PR DESCRIPTION
**Pull request by muarachmann**
This is based of https://github.com/luanpotter/audioplayers/pull/474 which was closed and useful for displaying next/previous icons on the system tray for IOS, I just rebased it and made it up-to date. All credits to @Pikaurd.

I have tested with various ios physical devices and it works  let me know if any changes are required. Here is my current flutter version

```
Flutter 1.17.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f7a6a7906b (4 weeks ago) • 2020-05-12 18:39:00 -0700
Engine • revision 6bc433c6b6
Tools • Dart 2.8.2
```

### CODE REVIEW

**Comment by muarachmann** [comment]
@luanpotter is there a reason for the build failing on ios-drive? If yes, I would love to know why thanks :)

**Comment by kofan** [comment]
Hey @muarachmann, great work! @luanpotter Is there any progress with this PR, are we going to merge it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +15,9 @@
 - Audio cache support for web
 - Re-adding partial web support
 
+## audioplayers 0.14.3
+- Add next and previous command for ios
```
Comment (luanpotter) [inline CHANGELOG.md:19]: duplicated entry

**Comment by muarachmann** [comment]
Opened a new PR in #598

**Pull request closed without merging**